### ISSUE TITLE
I changed my GitHub handle and it broke some things...

### ISSUE THREAD

**Issue by alexander-ding** [rsokl/Learning_Python#152]
Really sorry about this, but it seems like changing my GitHub handle (and hence changing the source 
hosting URL of the Chinese site) broke a few things. Now, it's hosted at http://alexander-ding.github.io/Learning_Python. My GitHub Pages setting still has custom domain as cn.pythonlikeyoumeanit.com, but navigating there doesn't work anymore. Is there perhaps some domain setting that you have to change to fix this? 

Also, the cn.pythonlikeyoumeanit.com was never setup with HTTPS. The GitHub Pages setting page complains that Enforce HTTPS is disabled because "Unavailable for your site because your domain is not properly configured to support HTTPS." I'm not really sure how to fix this. 

Finally, the link to my GitHub repo [here](https://www.pythonlikeyoumeanit.com/intro.html#contributors) is no longer accurate, though this is not that big a deal. I'll create a pull request at some point.

**Comment by alexander-ding** [comment]
Okay. The link seems to redirect just fine now, but the HTTPS part is still not working. It seems like it's the case for the English site as well. I did some checking, and [this thread](https://github.community/t/how-to-enable-https-support-on-custom-domains/10351/6) appears to provide the solution.

**Comment by rsokl** [comment]
@alexander-ding sorry for the very very delayed response. This came up just as BWSI was ramping up.

Is this still a matter of fixing HTTPS? I wonder why it works for the www site and not cn. I'll have to remind myself where I specified an IP address so that I can use one of the ones that you linked.

**Comment by alexander-ding** [comment]
Yeah. It seems like the www site's HTTPS is working just fine now. It wasn't when I last checked. I'm not sure what changed. The cn site is still not working though.
![image](https://user-images.githubusercontent.com/22578848/90030418-083c7980-dc8a-11ea-9368-ca060c2b565d.png)
